tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp

```diff
@@ -435,6 +435,9 @@ namespace tcf {
         JsonSetStr(resp, "jsonrpc", "2.0", "failed to serialize jsonrpc");
         JsonSetNumber(resp, "id", json_request_id, "failed to serialize json request id");
 
+        JsonSetStr(resp, "workOrderId", work_order_id.c_str(),
```

Comment:
"workOrderId" should be inside "data". It should be like
{
"jsonrpc": "2.0",
"id": <jrpc_request_id>,
"error": {
"code": <error_code>,
"message": <error_message>,
"data": {
"workOrderId": <work_order_id
}
}

Reply:
Done